### ISSUE TITLE
[Bug]: Import csv file delimited by other symbolic(except `,`), parse error.

### ISSUE THREAD

**Issue by chrysanthemum-boy** [infiniflow/infinity#729]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

main

### Commit ID

c4042976

### Actual behavior

Import data with delimited by blank_space, paser error.
```
terminate called after throwing an instance of 'infinity::ParserException'
  what():  Parse Integer error @src/parser/type/data_type.cpp:500
Aborted (core dumped)
```

```
terminate called after throwing an instance of 'infinity::ParserException'
  what():  Parse Float error @src/parser/type/data_type.cpp:526
Aborted (core dumped)
```
```
terminate called after throwing an instance of 'infinity::ParserException'
  what():  Boolean type should be true or false @src/parser/type/data_type.cpp:467
Aborted (core dumped)
```

### Steps to reproduce

```Markdown
Function test code can be found in python/test/test_import.py`test_with_blank_space_delimiter_data` `test_with_semicolons_delimiter_data`
```

**Comment by JinHai-CN** [comment]
Fixed by #824

**Comment by chrysanthemum-boy** [comment]
It looks fixed.
